performance_regression_row_level_repair_test.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2016 ScyllaDB
+
+import random
+import string
+import time
+
+from avocado import main
+from avocado.utils import process
+
+from sdcm.tester import ClusterTester
+
+# performance_regression_row_level_repair_test.py
+class PerformanceRegressionRowLevelRepairTest(ClusterTester):
+    """
+    Test Scylla performance regression with cassandra-stress.
+
+    :avocado: enable
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(PerformanceRegressionRowLevelRepairTest, self).__init__(*args, **kwargs)
+
+    # row-level-repair Test - measuring node-repair-time
+    def test_row_level_repair(self):
+        """
+        Test steps:
+
+        1. TODO: docstring
+        """
+        KEYSPACE_NAME = "ks"
```

Comment:
convention for variable names: upper case globals (module, class)
move to class level or make them lower case

Reply:
done